src/markdownPreview/markdownPreviewEdit.js

```diff
@@ -1,10 +1,15 @@
 'use strict';
 
 angular.module('adf.widget.scm')
-  .controller('MarkdownPreviewEditController', function (repositories, SCM) {
+  .controller('MarkdownPreviewEditController', function (repositories, SCM, $scope, $sce) {
     var vm = this;
     vm.repositories = repositories;
 
+    // tooltips
+    $scope.repositoryTooltip = $sce.trustAsHtml('Select the repository which stores the desired Markdown.');
```

Comment:
Please refer to voice & tone when writing messages. “A valid path to a markdown file must be stored here” might be rewritten to “Enter the path to a markdown file”.
Add information if the format of the path has to follow a special format (no leading / for example)